projects/1.18/assets/createaddition/createaddition/lang/zh_cn.json

```
@@ -1,5 +1,6 @@
 {
   "createaddition.tooltip.energy.stored": "已存储的能量：",
+  "createaddition.tooltip.energy.capacity": "容量:",

```

```suggestion
  "createaddition.tooltip.energy.capacity": "容量：",
```
标点符号应使用规范的汉语全角标点符号，具体要求请阅读我们的 [翻译规范](https://cfpa.site/TransRules/) 中 **03.标点符号** 这一节。
诸如这类的问题请在您翻译的所有文件中全部查明并修改。